New and modified procedures for new Hosts page. (#1033)
theforeman/foreman-documentation: guides/common/modules/proc_adding-errata-to-a-host.adoc

[id="Adding_errata_to_a_single_host_{context}"]
= Adding errata to a single host

Use the following procedure to add errata to a host.

.Procedure
. In the Satellite web UI, navigate to *Hosts > All Hosts* and select the host you want to add Errata to.
. On the new Host page, select the *Content* tab.
. On the *Content* page select the *Errata* tab.
. Select the errata you want to add to the host or select the checkbox at the top of the list to add all installable errata.
Click the check box next to any errata you wish to remove from a full list.

> . Using the three dots icon of the errata you want to add to the host, select *Apply via Katello agent* if UNKNOWN, *Apply via Remote Execution* if UNKNOWN, or *Apply via customized remote execution* if UNKNOWN.

Reviewer: katello agent and remote execution are two different technologies that can be used to achieve the task. REX is preferred, but katello-agent is the only option for users, who don't have connectivity from Capsule to the target host via SSH. The Customized REX mean, it will use REX but will allow user to customize the execution, e.g. set a specific time when it should be applied.

[id="Adding_errata_to_a_single_host_{context}"]
= Adding errata to a single host

Use the following procedure to add errata to a host.

.Procedure

> . In the Satellite web UI, navigate to *Hosts > All Hosts* and select the host you want to add Errata to.

Reviewer 2: ```suggestion
. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the host you want to add Errata to.
```

Reviewer 3: Please do not submit PRs without running checks locally for different scenarios. Every time we have to correct variables missing from your PRs. It's a waste of our time.

> . On the new Host page, select the *Content* tab.
> . On the *Content* page select the *Errata* tab.

Reviewer 2: ```suggestion
. Click the *Content* button and select the *Errata* tab.
```

I am unsure about this. IMHO "content" is really a button.

. Select the errata you want to add to the host or select the checkbox at the top of the list to add all installable errata.
Click the check box next to any errata you wish to remove from a full list.

> . Using the three dots icon of the errata you want to add to the host, select *Apply via Katello agent* if you have no connectivity to the target host using SSH.

Reviewer 2: IMHO we should not recommend users using "Katello Agent" any longer.
Maybe change the order and list REX first? Maybe add `(Deprecated)` to it? Or maybe even leave it out for now, because this is docs for Foreman 3.2+.
